lib/models/workout.dart

```diff
@@ -1,5 +1,28 @@
+import 'dart:convert';
+import 'package:http/http.dart' as http;
+
 import 'package:workoutpersonalizer_frontend/models/exercise.dart';
 
+Future<List> fetchWorkouts() async {
+  final response = await http.get(Uri.parse(
+      'http://localhost:8000/users/b70820ae-d0a3-411b-9217-0bf2370e7139/workouts/detailed'));
```

Comment:
Are we going to be using a local db? Or are we going to use a deployed version? If the latter is the case, can we make the `localhost:8000` a constant so that we can easily change it to point to the deployed db?

Also, I don't think we want to hardcode the UUID when calling the endpoint..

Reply:
This PR is for connecting the Workouts BE and FE. Kim made a separate PR for connecting the Exercises BE and FE. She's doing it the same way I am, hard coding the links so for now I think we should leave it. 

We can just do a Find and Replace later to fix it once all of these API calls are on main together, to deal with all of them at once.